support/support.sh

```diff
@@ -101,6 +112,20 @@ while read containerID status; do
     printf "\n"
 done < <(${DOCKER} container ls -a --format '{{.ID}} {{.Status}}' |cut -d' ' -f1,2)
 
+if [ "true" == ${SSD} ] ; then
+    echo "" ; echo "#### SSD control-plane and datapath consistency check on a node ####"
+    for netName in $(docker network ls -f driver=overlay --format "{{.Name}}") ; do
+        echo "## $netName ##"
+        ${SSDBIN} $netName
+        echo ""
+    done
+    echo "" ; echo "#### SSD control-plane consistency check across nodes in a cluster ####"
```

Comment:
@quadespresso let's take `gossip-consistency` out for now, we'll add it back once we address the caller